src/reducers/auth.lua

```diff
@@ -1,26 +1,33 @@
 local e = _G.PolarisNav
 
+local save = e.op.persistent_save
+local load = e.op.persistent_load
+
 local function set_auth(uid, tok, ses, code, att)
 	local auth = {
-		UserId = uid;
+		UserId = tonumber(uid);
```

Comment:
This is likely a trickled down effect from accidently converting numbers in the persistent_save function. Let's fix it at the root instead of band-aiding it.

Reply:
We should be able to remove this band-aid now
```suggestion
		UserId = uid;
```

Reply:
It would be fine elsewhere, but authentication code should only have the minimal logic and no unexpected behaviors (like random type conversions from an old bug).